src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -58,13 +58,24 @@ function OptionRowLHN({reportID, isFocused = false, onSelectRow = () => {}, opti
             : [styles.chatLinkRowPressable, styles.flexGrow1, styles.optionItemAvatarNameWrapper, styles.optionRow, styles.justifyContentCenter],
     );
 
-    if (!optionItem) {
+    if (!optionItem && !isFocused) {
```

Comment:
```
    if (!optionItem) {
         if (!isFocused) {
               return <View style={sidebarInnerRowStyle} />; 
         }
         return null;
   }
```

Reply:
I think it will be more readable

Reply:
I am a little skeptical about two if-statements 😄  How about if we extract this to a variable, like so:

```suggestion
    const isInitialRender = !optionItem && !isFocused;
    if (isInitialRender) {
```